server/python/README.md

```diff
@@ -4,19 +4,19 @@ This demo uses a simple [Flask](http://flask.pocoo.org/) application as the serv
 
 ## Payments Integration
 
-* [`app.py`](app.py) contains the routes that interface with Stripe to create charges and receive webhook events.
```

Comment:
We are technically not creating charges in this file anymore. Confusing?

Reply:
Yep, we forgot to update this README I'll go ahead and push a commit up